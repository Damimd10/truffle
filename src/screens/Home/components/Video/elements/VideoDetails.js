import styled from 'styled-components';

const VideoDetails = styled.section`
  padding: 1rem 0.5rem;
`;

VideoDetails.displayName = 'VideoDetails';

export default VideoDetails;
