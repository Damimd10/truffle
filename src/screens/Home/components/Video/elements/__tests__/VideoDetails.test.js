import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import VideoDetails from '../VideoDetails';

describe('Button Style Element', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<VideoDetails />);
  });

  it('should create a style structure for the component', () =>
    expect(toJson(wrapper)).toMatchSnapshot());
});
