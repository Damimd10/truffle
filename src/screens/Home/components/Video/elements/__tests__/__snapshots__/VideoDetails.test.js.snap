// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Style Element should create a style structure for the component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  padding: 1rem 0.5rem;
",
        ],
      },
      "displayName": "VideoDetails",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "section",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
